pb-test/src/bench.rs

```diff
@@ -94,7 +94,7 @@ mod benches {
 
 #[cfg(all(test, feature = "bench_prost"))]
 mod prost {
-    use bytes::Bytes;
+    use prost::bytes::Bytes;
```

Comment:
prost reexports bytes. Without this, it might stop compiling if the version of bytes brought in by prost doesn't match the one pb-test specifies.